### ISSUE TITLE
Oneliner sometimes duplicates submitted entries

### ISSUE THREAD

**Issue by AMcBain** [pouetnet/pouet2.0#85]
Normally posting twice in a row as by the same user leads to a "no double post; rob == jarig" type error page. However somehow sir got a double post. I don't think it was two separate attempts, based on the timestamps and the content, but a double execute of the submission.

Page showing the bug:
https://www.pouet.net/oneliner.php?page=9808

**Comment by AMcBain** [comment]
Attached screenshot of issue for posterity:
![screenshot_2019-02-08 oneliner pouet net](https://user-images.githubusercontent.com/83529/52475833-a5789800-2b51-11e9-9b1c-c28af6be0026.png)

**Comment by Gargaj** [comment]
Yeah SQL transactionality would fix this but I don't think it's worth the effort.
